fakturogen-invoices/src/main/java/pl/fakturogen/fakturogeninvoices/model/Customer.java

```diff
@@ -1,4 +1,38 @@
 package pl.fakturogen.fakturogeninvoices.model;
 
+import javax.persistence.*;
+
+@Entity
+@Table(name = Customer.TABLE_NAME)
 public class Customer {
+    public static final String TABLE_NAME = "customers";
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "customer_tax_number", nullable = false)
+    private String customerTaxNumber;
+
+    @Column(name = "customer_code", nullable = false)
+    private String customerCode;
+
+    @Column(name = "mail", nullable = false)
+    private String mail;
+
+    @Column(name = "phone_number")
+    private String phoneNumber;
+
+    @Enumerated
+    private CustomerType customerType;
+
+    @OneToOne
+    private Address address;
+
```

Comment:
same here: getter/setter etc.

Reply:
also here getters/setters are added (see 39 line and below)